server/events/comment_parser.go

```diff
@@ -31,18 +31,20 @@ import (
 )
 
 const (
-	workspaceFlagLong          = "workspace"
-	workspaceFlagShort         = "w"
-	dirFlagLong                = "dir"
-	dirFlagShort               = "d"
-	projectFlagLong            = "project"
-	projectFlagShort           = "p"
-	policySetFlagLong          = "policy-set"
-	policySetFlagShort         = ""
-	autoMergeDisabledFlagLong  = "auto-merge-disabled"
-	autoMergeDisabledFlagShort = ""
-	verboseFlagLong            = "verbose"
-	verboseFlagShort           = ""
+	workspaceFlagLong            = "workspace"
+	workspaceFlagShort           = "w"
+	dirFlagLong                  = "dir"
+	dirFlagShort                 = "d"
+	projectFlagLong              = "project"
+	projectFlagShort             = "p"
+	policySetFlagLong            = "policy-set"
+	policySetFlagShort           = ""
+	autoMergeDisabledFlagLong    = "auto-merge-disabled"
+	autoMergeDisabledFlagShort   = ""
+	verboseFlagLong              = "verbose"
+	verboseFlagShort             = ""
+	clearPolicyApprovalFlagLong  = "clear-approval"
```

Comment:
I think this needs to be more concise like `policy-clear-approval` since clear approval could be for applies too nd could get confusing